src/Lexer.js

```diff
@@ -75,14 +75,14 @@ const tokenRegex = new RegExp(
     "([ \r\n\t]+)|" +                                 // whitespace
     "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" +  // single codepoint
     "|[\uD800-\uDBFF][\uDC00-\uDFFF]" +               // surrogate pair
-    "|\\\\(?:[a-zA-Z]+|[^\uD800-\uDFFF])" +           // function name
+    "|\\\\(?:[a-zA-Z@]+|[^\uD800-\uDFFF])" +          // function name
```

Comment:
Good to have at least one negative reaction -- I was curious what others would think.  I see three alternatives:

1. Add a `Lexer` option for whether to include `@` as a letter.  Then we could actually support `\makeatletter` and `\makeatother` (presumably this would need to be carried in the `Settings` object...).
2. Have the `cdots` macro return manually parsed `Token`s instead of strings.  Then we don't need to touch the `Lexer`.
3. Rename `\@cdots` to something else like `\cdotsINTERNAL` or `\latexcdots`.

Thoughts/preferences?